aten/src/ATen/native/BinaryOps.h

```diff
@@ -7,6 +7,23 @@ namespace at { struct TensorIterator; }
 
 namespace at { namespace native {
 
+static inline void alpha_check(const ScalarType dtype, Scalar alpha) {
```

Comment:
These should not be static; static means that these functions are not used outside of their translation unit and is used for helper functions in .cpp files

Reply:
removed `static`